lib/taskana-core/src/main/java/pro/taskana/task/internal/ServiceLevelHandler.java

```diff
@@ -48,15 +46,7 @@
     this.taskanaEngine = taskanaEngine;
     this.taskMapper = taskMapper;
     this.attachmentMapper = attachmentMapper;
-    DaysToWorkingDaysConverter.setGermanPublicHolidaysEnabled(
-        taskanaEngine.getEngine().getConfiguration().isGermanPublicHolidaysEnabled());
-    try {
-      converter = DaysToWorkingDaysConverter.initialize();
-    } catch (InvalidArgumentException e) {
-      LOGGER.error(ERROR_CANNOT_INITIALIZE_DAYS_TO_WORKING_DAYS_CONVERTER);
-      throw new SystemException(
-          ERROR_CANNOT_INITIALIZE_DAYS_TO_WORKING_DAYS_CONVERTER, e.getCause());
-    }
+    this.converter = WorkingDaysToDaysConverter.initialize();
```

Comment:
Why don't we move this to the engine as well?

Reply:
The engine calls only WorkingDaysToDaysConverter.setGermanPublicHolidaysEnabled() which is a static method. The initialize() call is a factory method that creates an instance of WorkingDaysToDaysConverter.
Since the WorkingDaysToDaysConverter is only needed inside ServiceLevelHandler, I made it an instance variable of this class.
To make things consistent, I'll also move the setGermanPublicHolidaysEnabled() call to the ctor of ServiceLevelHandler.